### PR TITLE
switch uses of old button toolkit template to digitalmarketplace-govuk-frontend

### PR DESCRIPTION
**Pull request by risicle**
https://trello.com/c/reSSXWv8/

Took a bit of tinkering to be able to see all of these uses, but everything I found looked ok.

### CODE REVIEW

File: app/templates/direct-award/save-search.html [hunk 1]
```diff
@@ -87,7 +87,11 @@ <h1>
 
         <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
         <input type="hidden" name="search_query" value="{{ search_query }}"/>
-        <input type="submit" class="button-save" name="return_to_overview" value="Save and continue">
```
Comment (aliuk2012) [inline app/templates/direct-award/save-search.html]: good catch

File: app/templates/search/_filters.html [hunk 2]
```diff
@@ -13,5 +13,11 @@
...
     {% endfor %}
   </div>
-  <button class="button-save js-hidden js-dm-live-search" type="submit" aria-controls="search-summary-accessible-hint-wrapper">Filter</button>
+  {{ govukButton({
+    "text": "Filter",
+    "classes": "js-hidden js-dm-live-search",
+    "attributes": {
+      "aria-controls": "search-summary-accessible-hint-wrapper",
+    },
+  }) }}
```
Comment (katstevens) [inline app/templates/search/_filters.html]: Functional tests seem happy with this 👍

Comment (risicle) [reply app/templates/search/_filters.html]: Yeah I think for button clicking we mostly use a fairly universal selector.

File: app/templates/direct-award/end-search.html [hunk 3]
```diff
@@ -65,13 +65,18 @@ <h1>
 
         {{ form.user_understands }}
 
-        <button class="button-save end-your-search-button"
```
Comment (aliuk2012) [inline app/templates/direct-award/end-search.html]: Another good catch

File: app/templates/search/_services_save_search.html [hunk 4]
```diff
@@ -1,15 +1,21 @@
+{# this file can be used from contexts that don't inherit from _base_page.html, so we must ensure we have all imports we need #}
+{% from "components/button/macro.njk" import govukButton %}
```
Comment (aliuk2012) [inline app/templates/search/_services_save_search.html]: 😭, thanks for documenting this partial and explaining why an import was needed here.

**Comment by risicle** [comment]
Looks like I missed at least one old-style button in this... https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/aba67473d3b41bd731c665cca39c194b80c684ae/app/templates/brief.html#L149

**Pull request merged**